tests/client.rs

```diff
@@ -97,17 +97,20 @@ async fn http3_request_full() {
     });
 
     let url = format!("https://{}/content-length", server.addr());
-    let res = reqwest::Client::builder()
+    let res_fut = reqwest::Client::builder()
         .http3_prior_knowledge()
         .danger_accept_invalid_certs(true)
         .build()
         .expect("client builder")
         .post(url)
         .version(http::Version::HTTP_3)
         .body("hello")
-        .send()
-        .await
-        .expect("request");
+        .send();
+
+    fn assert_sync<T: Sync>(_: &T) {}
+    assert_sync(&res_fut);
```

Comment:
IMHO you could also put the `Send` bound as well

```suggestion
    fn assert_send_sync<T: Send + Sync>(_: &T) {}
    assert_send_sync(&res_fut);
```

I think you could make it a global function?

Reply:
changed it to a global function for `Send` and `Sync`